third_party/tsl/tsl/platform/default/rocm_rocdl_path.cc

```diff
@@ -40,6 +40,13 @@ string RocmRoot() {
 #endif
 }
 
-string RocdlRoot() { return io::JoinPath(RocmRoot(), "amdgcn/bitcode"); }
+string RocdlRoot() {
```

Comment:
Can I ask you to use the opportunity to replace this by `std::string`?

Reply:
good idea! thanks for the suggestion!